### DISCUSSION TITLE
Allow creation of silence from an alert

### DISCUSSION THREAD

**Discussion by Hooloovoo** [grafana/grafana#56610] [Alerting]
I would like to be able to create a silence from a firing alert, so that it takes me to the "New Silence" screen with as much information pre-populated as possible that would capture the relevant alert as specifically as possible. I could then delete some of the labels etc if I wanted to make the silence more generic, but it would save a lot of copying and pasting.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
